launcher/game/EasyDialogsWin.py

```diff
@@ -673,10 +673,10 @@ def hookProc(hdlg, uiMsg, wParam, lParam):
     ofn.lpfnHook = LPOFNHOOKPROC(hookProc)
 
     if fn(ctypes.byref(ofn)):
-        filenames = filter(None, filename.split('\0'))
+        filenames = [fn for fn in filename.split('\0') if fn]

```

Comment:
I think reusing the identifier `fn` when it is used as a function earlier is confusing.

Reply:
Updated.